flex/interactive/sdk/python/gs_interactive/client/session.py

```diff
@@ -439,6 +439,11 @@ def get_graph_schema(
         self,
         graph_id: Annotated[StrictStr, Field(description="The id of graph to get")],
     ) -> Result[GetGraphSchemaResponse]:
+    #Get the schema of a specified graph.
```

Comment:
Follow the existing comment style.

https://github.com/alibaba/GraphScope/blob/main/python/graphscope/analytical/app/average_degree_connectivity.py#L32